model/wire.cc

```diff
@@ -103,6 +103,7 @@ namespace minsky
   {
     auto f=from(), t=to();
     if (attachedToDefiningVar()) return false;
+    if (f->item().attachedToDefiningVar()) return false;      // ensure wires attached to out port do not dangle. for ticket 1275
```

Comment:
Maybe for wires, visibility is simply the conjunction (&&) of the visibility of the to() and from() objects.

Reply:
I tried these combinations but they lead to segfaults in Wire::attachedToDefiningVar() and more dangling wires in Wire::visible().

Reply:
Maybe you're conflating visibility with attachedToDefiningVar...

Reply:
Yes, I think we should discuss this on Monday morning during our meeting. I think I'm missing the point of the feature. I thought it was to reduce clutter on the canvas by moving a var to the tab.